byway/src/main/webapp/recommendations.js

```diff
@@ -201,7 +201,7 @@ function placeMarker(place) {
     content: place.name
   });
   const image = {
-    url: '/images/stopsMarker.png',
```

Comment:
rooted absolute URL is better, since it won't depend on where the page is located relative to the image.

Reply:
fixed